internal/jennies/java/templates/types/class.tmpl

```diff
@@ -28,6 +28,14 @@ public class {{ .Name }}{{ if .Extends }} extends {{ range $i, $e := .Extends }}
         return {{ $.Name | lowerCamelCase }};
     }
     {{- end }}
+    {{- else if .DefaultConstructorArgs }}
+    public {{ $.Name }}() {}
```

Comment:
What is this empty constructor useful for?
If some fields in the object are required, it would allow constructing an object in an "invalid" state

Reply:
When you create a constructor, the default disappears unless you define it. I added to avoid to refactor the builder's code to use the new ones. We could eventually do it to make the code a bit nicer.